src/libraries/Microsoft.PowerFx.Transport.Attributes/TransportConverterAttribute.cs

```diff
@@ -2,14 +2,16 @@
 // Licensed under the MIT license.
 
 using System;
+using System.Diagnostics.CodeAnalysis;
 
 namespace Microsoft.AppMagic.Transport
 {
     /// <summary>
     /// Applied to a converter class. Provides a conversion for a custom type.
     /// </summary>
     [AttributeUsage(AttributeTargets.Class)]
-    public class TransportConverterAttribute : Attribute
+    [SuppressMessage("Design", "CA1019: Define accessors for attribute arguments", Justification = "n/a")]
+    public sealed class TransportConverterAttribute : Attribute
```

Comment:
technically, these are a breaking change, but it should be safe. I don't think any uses in PA-Client derive from any of these attributes, and I'd be really surprised if someone else relied on this package.

Reply:
agreed, I don't expect any problem with this change